include/dev/strainGauge.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef strainGauge_STRAINGAUGE_HPP
+#define strainGauge_STRAINGAUGE_HPP
+
+#include <EVT/io/ADC.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace SGM::DEV {
+
+class strainGauge {
```

Comment:
This class and the corresponding file names should be UpperCamelCase
```suggestion
class StrainGauge {
```